libraries/cbor/src/macros.rs

```diff
@@ -608,30 +595,26 @@ mod test {
     }
 
     #[test]
-    fn test_cbor_map_btree_empty() {
-        let a = cbor_map_btree!(BTreeMap::new());
-        let b = Value::Map(BTreeMap::new());
+    fn test_cbor_map_collection_empty() {
+        let a = cbor_map_collection!(BTreeMap::new());
```

Comment:
Why not `Vec::new()`?

Reply:
We can do both, if we support initializing from `BTreeMap` and `Vec`. See below.